hello/hello_activity.py

```diff
@@ -1,27 +1,36 @@
 import asyncio
 import logging
+from dataclasses import dataclass
 from datetime import timedelta
 
 from temporalio import activity, workflow
 from temporalio.client import Client
 from temporalio.worker import Worker
 
 
+@dataclass
+class ComposeGreetingInput:
```

Comment:
You should probably document why the sample uses a dataclass and not multiple activity params

Reply:
I will do it here. I hope you are not suggesting I have to do this everywhere I use a data class forever.

Reply:
Done